lib/power_action_utils.pm

```diff
@@ -49,20 +49,26 @@ $vnc_console get required variable 'SVIRT_VNC_CONSOLE' before assignment.
 =cut
 sub prepare_system_shutdown {
     # kill the ssh connection before triggering reboot
-    console('root-ssh')->kill_ssh if get_var('BACKEND', '') =~ /ipmi|spvm|pvm_hmc/;
-
-    if (check_var('ARCH', 's390x')) {
+    if (get_var('BACKEND', '') =~ /ipmi|spvm|pvm_hmc/) {
+      console('root-ssh')->kill_ssh if get_var('BACKEND', '') =~ /ipmi|spvm|pvm_hmc/;
+      record_info('prepare_system_shutdown: ssh connection is disconnected');
```

Comment:
@SergioAtSUSE For both lines, this could be a good message, so they're disting
This should make the record_info distinguishable enough
```suggestion
      record_info('prepare_system_shutdown: ssh connection is disconnected on backend: '.get_var(BACKEND,'NONE'));
```